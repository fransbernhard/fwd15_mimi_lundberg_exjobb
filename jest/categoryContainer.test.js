import React, { Component } from 'react';
import renderer from 'react-test-renderer';
import CategoryContainer from '../src/components/categoryContainer';

describe('components', () => {
  describe('<CategoryContainer/>', () => {
    it('renders correctly', () => {
      const tree = renderer
        .create(
          <CategoryContainer
            filterHandler={jest.fn}
            products={[
              {category: "fruit"},
              {category: "fish"}
            ]}
          />
        )
        .toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
