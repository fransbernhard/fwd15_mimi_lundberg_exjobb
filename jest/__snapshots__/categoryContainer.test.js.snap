// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <CategoryContainer/> renders correctly 1`] = `
<div>
  <ul
    className="filterList"
  >
    <div>
      <li
        className="category"
        onClick={[Function]}
      >
        fruit
      </li>
    </div>
    <div>
      <li
        className="category"
        onClick={[Function]}
      >
        fish
      </li>
    </div>
  </ul>
</div>
`;
